.github/workflows/ci.yml

```diff
@@ -21,13 +21,20 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Build and Push Docker Image
-        uses: ./.github/actions/build
+        uses: Dwolla/jenkins-agents-workflow/.github/actions/build@main
```

Comment:
This docker-java repo isn't a Jenkins agent. Can you describe how this works here?

Reply:
I think this comment may have been resolved in a slack thread, but I'll post out here in case anyone else has the same question.

We can certainly rename `jenkins-agents-workflow` (it will just require a lot of updating in all of the caller workflows). 90+ percent of the caller workflows to jenkins-agents-workflow are jenkins agents, so I think the name "jenkins agents" makes sense for now.

For background, when I started writing jenkins-agents-workflow I wasn't planning on replacing this code here. As things started to split out into almost exactly the same composite action we have here, though, I thought it would make sense to reduce duplicate code and call the build action housed in the `jenkins-agents-workflow` repo.